misc/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+# Create your models here.
+class BaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+class Meta:
```

Comment:
Meta is a nested classes for BaseModel. Please use indentation to make it nested.